cmd/preview/cmdpreview.go

```diff
@@ -114,10 +113,16 @@ func (r *PreviewRunner) RunE(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
+		namespace, enforceNamespace, err := r.factory.ToRawKubeConfigLoader().Namespace()
```

Comment:
Maybe a comment here for those not familiar with namespace within the kubeconfig file.